### ISSUE TITLE
Fotorama Gallery too sensitive on Android Devices. (chrome)

### ISSUE THREAD

**Issue by joebordo** [magento/magento2#7906]
**2 Problems because of Chrome Update.**

I think this happend after Chrome update...(something changed in chrome)
Everything used to work fine...i think until couple of days ago...i noticed my chrome got updated on my android phone and so seems to be my pc chrome. 

### Preconditions
1.  Magento 2.1.3, 2.1.2, 2.1.1
2. Use Chrome Dev tools=>mobile view, with touch, or Chrome in Android Phone

### Steps to reproduce
You can even use this demo page.
Go with your android mobile, or google developer tools, mobile view to the following page.
http://magento2-demo.nexcess.net/radiant-tee.html

Edit: This link has more images.: 
http://alothemes.com/demo2/supermarket/index.php/favorite-izon-cameras.html

So basycally all themeforest themes...can be checked...

Then swipe right or left. The image swipes instantly. You can't grab the image, hold it and control it. 
It used to work normal before, but i think after chrome update...things changed.
It's bad because if you try to scroll down, it always triggers the swipe...
It also feels that you are not the one swiping.

On Iphone it works fine. ON my LG G4 "internal browser"=not chrome" it also works fine.
Alot of visitors use chrome...so it's really important.
### Expected result
**Problem 1**
You should be able to swipe hold it, and let it go. To fully contorl your swipes like before.
It should not trigger a swipe when you scroll down.
**Problem 2**
The thumbnav slider becomes really hard to scroll. Like it has alot of friction. 
### Actual result
**Problem 1**
It triggers the swipe and makes it look messy. Just keep scrolling down on the image...it keeps getting triggered. While being scrolled it is also being swiped.
**Problem 2**
The user should be able to scroll easy...and smooth.

PS: I just downloaded latest fotorama 4.6.4 uncompressed and replaced it with my magento fotorama.js in lib folder.
That one works! No problem at all...but it's not fully coded with magento variables...the magento fotorama.js has been modified to work properly with magento.

So the problem is with the fotorama.js of magento.

If you can"t replicate the problem, let me know i will make a video and put it on youtube.

Ps: This is only on Android, chrome which alot of users use. I tried on safari and chrome on ipad and everything seems fine.
I hope someone can find a fix.

**Comment by joebordo** [comment]
Coman guys, anyone? I can't sleep. It's really bugging me. I don't know what to do.

Edit: So what i found out is really weard. Fotorama had this scroll down problem right? Now using the old "fotorama of 2.0 or 2.1" without the fix for it to know if you wanna scroll down or swipe...it works.

So it's defently the chrome update.

It's still a mystery why the original from fotorama.io behaves normal.

**Comment by ghost** [comment]
We replaced fotorama with magic zoom. Less issues on mobile. Just a suggestion if you cant sort your issue.

**Comment by joebordo** [comment]
The problem is not fotorama it self , but the way they implemented the gallery.
I'm now going 1 line by 1 line over 3000 lines with the original fotorama.js to see what they did different to find out why magento version behaves the way it does, but not the original.

and i'm not a javascript developer lol. I know it's a tiny easy peacy thing...but no one except you replies :(

Thanks !

**Comment by joebordo** [comment]
Okey, found it. 

**Line: 1508**

```
  $el.on('mousedown pointerdown', onStart);
            $DOCUMENT
                .on('mousemove pointermove', onMove)
                .on('mouseup pointerup', onEnd);
        }
```

Get rid of the "pointerDown en pointermove stuff...

So it becomes.

```
     $el.on('mousedown', onStart);
            $DOCUMENT
                .on('mousemove', onMove)
                .on('mouseup', onEnd);
        }
```

**Comment by ghost** [comment]
You probably should of left this open if it hinders user interaction. Could be something the dev team needs to look into. In any event, glad you found a solution.

**Comment by joebordo** [comment]
You are right. I just reopened.Thanks! Hope it helps others.

**Comment by magento-engcom-team** [comment]
@joebordo, thank you for your report.
We've created internal ticket(s) MAGETWO-83114 to track progress on the issue.

**Comment by magento-engcom-team** [comment]
Hi @joebordo. Thank you for your report. 
The issue has been fixed in magento/magento2#14123 by @DaniloEmpire in 2.3-develop branch
Related commit(s):
 - [166a8bb302f0165ef720bdc73db664f4617c8e7a](https://github.com/magento/magento2/commit/166a8bb302f0165ef720bdc73db664f4617c8e7a)
 - [159efeb46daaba9098b21d696249027d2b0c0bc7](https://github.com/magento/magento2/commit/159efeb46daaba9098b21d696249027d2b0c0bc7)
 - [50579a6171e6bf8f73771cdaafcc99574eb1cf44](https://github.com/magento/magento2/commit/50579a6171e6bf8f73771cdaafcc99574eb1cf44)

The fix will be available with the upcoming 2.3.0 release.

**Comment by ghost** [comment]
2.3 release date?

**Comment by Ilchuk** [comment]
#mageconf

**Comment by magento-engcom-team** [comment]
@Ilchuk thank you for joining. Please accept team invitation [here](https://github.com/orgs/magento/invitation) and self-assign the issue.

**Comment by magento-engcom-team** [comment]
Hi @TomashKhamlai. Thank you for working on this issue. 
Looks like this issue is already verified and confirmed. But if your want to validate it one more time, please, go though the following instruction:
- [x] 1. Add/Edit `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.

- [x] 2. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 3. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 4. If the issue is not relevant or is not reproducible any more, feel free to close it.

**Comment by TomashKhamlai** [comment]
The issue was re-tested and we can confirm that it was fixed on the 2.3 release branch. We closing this issue as fixed due to upcoming 2.3 release that will be available soon.

![2 3 0-magento-trim](https://user-images.githubusercontent.com/11827230/48553941-ad73c380-e8e5-11e8-8b8b-4e5d07522161.gif)

**Comment by magento-engcom-team** [comment]
Hi @joebordo. Thank you for your report. 
The issue has been fixed in magento/magento2#22078 by @Nazar65 in 2.2-develop branch
Related commit(s):
 - [744986b6b081c2f1e562ec25988a22a5e5970767](https://github.com/magento/magento2/commit/744986b6b081c2f1e562ec25988a22a5e5970767)

The fix will be available with the upcoming 2.2.9 release.
